assets/package.json

```diff
@@ -20,12 +20,15 @@
     "lint:check": "eslint --max-warnings 0 \"{.,**}/*.{js,json,ts,tsx}\"",
     "format": "prettier --write \"{.,**}/*.{js,json,ts,tsx,css,scss}\"",
     "format:check": "prettier --check \"{.,**}/*.{js,json,ts,tsx,css,scss}\"",
-    "cypress:open": "cypress open"
+    "cypress:open": "cypress open",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build"
   },
   "dependencies": {
     "@react-hook/media-query": "^1.1.1",
     "@rehooks/component-size": "^1.0.3",
     "@sentry/react": "^7.15.0",
+    "@storybook/preset-scss": "^1.0.3",
```

Comment:
**question:** can this go in `devDependencies` instead?

Reply:
Thanks for catching this! 7518ce2c2b906bb25b46869fef1ee0da101adbb3